### PR TITLE
Fix issue #48, settings.xml is generated incorrectly for windows. 

### PR DESCRIPTION
**Pull request by fredcooke**
The fix should be fine on Linux and Mac, too.

### CODE REVIEW

File: branchout-maven [hunk 1]
```diff
@@ -64,7 +64,7 @@ function mavenSettings() {
 <settings xsi:schemaLocation=\"http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd\" xmlns=\"http://maven.apache.org/SETTINGS/1.1.0\"
     xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\">
 
-  <localRepository>${BRANCHOUT_STATE}/maven/repository</localRepository>
+  <localRepository>\${user.home}/branchout/${BRANCHOUT_NAME}/maven/repository</localRepository>
```
Comment (stickycode) [inline branchout-maven:67]: as per discussion, we should just use cygpath to get the value correcct on windows

Comment (stickycode) [reply branchout-maven:67]: the problem is ultimately because branchout is unix based but the settings file is read from a native context i.e. windows

Comment (fredcooke) [reply branchout-maven]: Resolved and all checks passed, approve and if I can I'll push direct.

**Review by stickycode — APPROVED** [review]

**Comment by fredcooke** [comment]
Why did you merge bad code, @stickycode :-(

**Comment by fredcooke** [comment]
Fix coming in the next PR.

**Pull request merged**